README.md

```diff
@@ -57,9 +59,10 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
-
+- Cindy Shinon Behance for her UI design template
```

Comment:
- [ ] As mentioned by the previous reviewer you should add an "Acknowledgements" section or similar that gives appropriate credit to[Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), not mention it he but also add the links 🙃 😉